### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by kbagchiGWC**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+> Please ensure that you are not sharing any
+[Personally Identifiable Information(PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
+or sensitive account information (API keys, credentials, etc.) when reporting an issue.
```
Comment (charliesantos) [inline .github/ISSUE_TEMPLATE/bug_report.md:12]:
```suggestion
- [ ] I have verified that the issue occurs with the latest release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-voice-react-native/blob/main/CHANGELOG.md).
- [ ] I reviewed the [Common Issues](https://github.com/twilio/twilio-voice-react-native/blob/main/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
- [ ] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
       or sensitive account information (API keys, credentials, etc.) when reporting this issue.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+What actually happens.
+
+**Reproduces How Often**
+
+What percentage of the time does it reproduce?
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Smartphone (please complete the following information):**
```
Comment (charliesantos) [inline .github/ISSUE_TEMPLATE/bug_report.md:38]:
```suggestion
**Software and Device Information (please complete the following information):**
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+ - Device: [e.g. iPhone, Pixel3]
+ - OS: [e.g. iOS16, Android API Leveln 31]
+ - Version [e.g. 22]
```
Comment (charliesantos) [inline .github/ISSUE_TEMPLATE/bug_report.md:41]: What is "Version" in this context? The SDK version?

**Review by charliesantos — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (mhuynh5757) [reply .github/ISSUE_TEMPLATE/bug_report.md:41]: +1, let's ask for as much info as possible. React version, react native version, node version, npm or yarn version

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+**Reproduces How Often**
+
+What percentage of the time does it reproduce?
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Software and Device Information (please complete the following information):**
+ - Device: [e.g. iPhone, Pixel3]
+ - OS: [e.g. iOS16, Android API Leveln 31]
```
Comment (mhuynh5757) [inline .github/ISSUE_TEMPLATE/bug_report.md:41]:
```suggestion
 - OS: [e.g. iOS16, Android API Level 31]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+**Reproduces How Often**
+
+What percentage of the time does it reproduce?
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Software and Device Information (please complete the following information):**
+ - Device: [e.g. iPhone, Pixel3]
```
Comment (mhuynh5757) [inline .github/ISSUE_TEMPLATE/bug_report.md:40]:
```suggestion
 - Device: [e.g. iPhone, Pixel 3]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+What percentage of the time does it reproduce?
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Software and Device Information (please complete the following information):**
+ - Device: [e.g. iPhone 12, Pixel 3]
+ - OS: [e.g. iOS 16, Android API Level 31]
+ - React Version
```
Comment (mhuynh5757) [inline .github/ISSUE_TEMPLATE/bug_report.md:34]:
```suggestion
 - React version: [e.g. 18.1.0]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+ - React Native Version
```
Comment (mhuynh5757) [inline .github/ISSUE_TEMPLATE/bug_report.md:35]:
```suggestion
 - React Native version: [e.g. 0.70.9]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+ - Node Version
```
Comment (mhuynh5757) [inline .github/ISSUE_TEMPLATE/bug_report.md:36]:
```suggestion
 - Node version: [e.g. 16.18.1]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+ - npm or yarn version
```
Comment (mhuynh5757) [inline .github/ISSUE_TEMPLATE/bug_report.md:37]:
```suggestion
 - npm or yarn version: [e.g. 8.19.2, 1.22.19]
```

**Review by mhuynh5757 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
+
```
Comment (mhuynh5757) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: Can we add a title here?

```
# Submission Checklist
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+- [ ] I have verified that the issue occurs with the latest release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-voice-react-native/blob/main/CHANGELOG.md).
+- [ ] I reviewed the [Common Issues](https://github.com/twilio/twilio-voice-react-native/blob/main/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
+- [ ] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
+       or sensitive account information (API keys, credentials, etc.) when reporting this issue.
+
+**Describe the bug**
```
Comment (mhuynh5757) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
## Description of Issue
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```
Comment (mhuynh5757) [inline .github/ISSUE_TEMPLATE/bug_report.md:10]:
```suggestion
## Reproduction Steps
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+- [ ] I have verified that the issue occurs with the latest release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-voice-react-native/blob/main/CHANGELOG.md).
+- [ ] I reviewed the [Common Issues](https://github.com/twilio/twilio-voice-react-native/blob/main/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
+- [ ] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
+       or sensitive account information (API keys, credentials, etc.) when reporting this issue.
+
+**Describe the Issue**
+A clear and concise description of what the issue is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
```
Comment (mhuynh5757) [inline .github/ISSUE_TEMPLATE/bug_report.md:11]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 14]
```diff
@@ -0,0 +1,40 @@
...
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
```
Comment (mhuynh5757) [inline .github/ISSUE_TEMPLATE/bug_report.md:17]:
```suggestion
## Expected Behavior
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 15]
```diff
@@ -0,0 +1,40 @@
...
+A clear and concise description of what you expected to happen.
+
+**Actual Behavior**
```
Comment (mhuynh5757) [inline .github/ISSUE_TEMPLATE/bug_report.md:20]:
```suggestion
## Actual Behavior
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 16]
```diff
@@ -0,0 +1,40 @@
...
+
+What actually happens.
+
+**Reproduces How Often**
```
Comment (mhuynh5757) [inline .github/ISSUE_TEMPLATE/bug_report.md:24]:
```suggestion
## Reproduction Frequency
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 17]
```diff
@@ -0,0 +1,40 @@
...
+
+What percentage of the time does it reproduce?
```
Comment (mhuynh5757) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]:
```suggestion
Is the reproducibility of the issue deterministic? If not, what percentage of the time does the issue occur? In how many attempts was the issue observed?
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 18]
```diff
@@ -0,0 +1,40 @@
...
+
+**Screenshots**
```
Comment (mhuynh5757) [inline .github/ISSUE_TEMPLATE/bug_report.md:28]:
```suggestion
## Screenshots
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 19]
```diff
@@ -0,0 +1,40 @@
...
+If applicable, add screenshots to help explain your problem.
+
+**Software and Device Information (please complete the following information):**
```
Comment (mhuynh5757) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]:
```suggestion
## Software and Device Information
**Please complete the following information.**
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 20]
```diff
@@ -0,0 +1,40 @@
...
+ - Device: [e.g. iPhone 12, Pixel 3]
+ - OS: [e.g. iOS 16, Android API Level 31]
+ - React version: [e.g. 18.1.0]
+ - React Native version: [e.g. 0.70.9]
+ - Node version: [e.g. 16.18.1]
+ - npm or yarn version: [e.g. 8.19.2, 1.22.19]
+
+**Additional context**
```
Comment (mhuynh5757) [inline .github/ISSUE_TEMPLATE/bug_report.md:39]:
```suggestion
## Additional Context
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 21]
```diff
@@ -0,0 +1,40 @@
+
+- [ ] I have verified that the issue occurs with the latest release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-voice-react-native/blob/main/CHANGELOG.md).
+- [ ] I reviewed the [Common Issues](https://github.com/twilio/twilio-voice-react-native/blob/main/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
+- [ ] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
+       or sensitive account information (API keys, credentials, etc.) when reporting this issue.
+
+**Describe the Issue**
```
Comment (mhuynh5757) [inline .github/ISSUE_TEMPLATE/bug_report.md:7]:
```suggestion
## Description
```

**Review by mhuynh5757 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 22]
```diff
@@ -0,0 +1,40 @@
...
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+## Expected Behavior
+A clear and concise description of what you expected to happen.
+
+## Actual Behavior
+
```
Comment (mhuynh5757) [inline .github/ISSUE_TEMPLATE/bug_report.md:20]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 23]
```diff
@@ -0,0 +1,40 @@
...
+What actually happens.
+
+## Reproduction Frequency
+
```
Comment (mhuynh5757) [inline .github/ISSUE_TEMPLATE/bug_report.md:24]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 24]
```diff
@@ -0,0 +1,40 @@
+# Submission Checklist
```
Comment (mhuynh5757) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]:
```suggestion
# Issue

## Pre-submission Checklist
```

**Review by mhuynh5757 — CHANGES_REQUESTED** [review]

**Review by mhuynh5757 — APPROVED** [review]

**Review by charliesantos — APPROVED** [review]

**Pull request merged**